.storybook/stories/modal/modal-static.stories.ts

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016-2023 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { ClrModalModule } from '@clr/angular';
+import { Parameters } from '@storybook/addons';
+import { Story } from '@storybook/angular';
+
+import { setupStorybook } from '../../helpers/setup-storybook.helpers';
+
+const defaultStory: Story = args => ({
+  template: `
+    <style>
+      .backdrop-example-container {
+        position: relative;
+        padding: 24px;
+      }
+      .modal.static {
+        position: relative;
+        padding: 72px;
+      }
+        
+      .modal-backdrop.static {
+        position: absolute;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+      }
+    </style>
+    <div class="backdrop-example-container">
+        <div class="modal static">
+            <div class="modal-dialog modal-{{size}}" role="dialog" aria-hidden="true">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button aria-label="Close" class="close" type="button">
+                            <cds-icon aria-hidden="true" shape="window-close"></cds-icon>
```

Comment:
This and the other button still has double indentation. I will approve this though and you can fix it if you want.

Reply:
Had a wrong configuration for spaces. Should be good now :)